mantidproject/mantid
# - Temporarily hide plot results checkbox by default to prevent issues during ISIS SANS merge reduction

### Description of work
When performing merged reduction, if both the scale option and the plot result option are selected, it causes an issue. 

#### Summary of work
- The issue is resolved by hiding the plot result checkbox, and also adding an option for Users to enable the visibility of the plot result checkbox from their properties file.
- the properties file documentation is updated with instructions on how to enable or disable the plot-results checkbox
- the ISIS-SANS documentation is updated as well to reflect the new changes 
This fix will help us determine whether to permanently hide the checkbox or fix the underlying issue.

Fixes: #37577

### To test:
1) Go to `Interfaces`->`SANS`->`ISIS SANS`
2) It will be noticed that the plot result checkbox is hidden
3) To show the plot result checkbox,  `sans.isis_sans.plotResults=On` should
be added to your mantid.user.properties file, 
4) Close and re-run Mantid, then repeat step 1
5) The `Plot Results` check box will appear at the bottom right of the interface.
6) To hide the plot result checkbox, `sans.isis_sans.plotResults=Off` should
be added to your mantid.user.properties file or left empty
7) For Further Testing: At the top of the Runs page, load the user file `MaskFile.toml` from the loqdemo folder in the TrainingCourseData.
6) Load the `batch_mode_reduction.csv` file from the loqdemo folder in the TrainingCourseData.
7) Navigate to the Settings page from the left hand menu. Under the `General, Scale, Event Slice, Sample` tab change the following:
      - Set the `Reduction Mode` dropdown at the top to `merged`.
      - In the `Merged Settings` section at the top, tick the `Use fit` checkbox next to the `Scale` field.
8) Use the left hand menu to navigate back to the Runs page.
9) Ensure `Plot Results` checkbox is hidden
10) The reduction will complete successfully with no errors

### Documentation Test 
1) Build developer documentation with ninja dev-docs-html
2) Go to Runs Tab page
3) Scroll to the bottom and check that the plot result column contains information on the changes made
4) Go to Properties File  page, scroll to the bottom, check that the page contain instructions on how to 
hide or show the plot result checkbox from mantid.user.properties file.
